packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -842,7 +855,10 @@ class ThemeData extends Diagnosticable {
         chipTheme,
         platform,
         materialTapTargetSize,
-        cursorColor
+        hashValues(
+          pageTransitionsTheme,
+          cursorColor,
+        ),
```

Comment:
please make sure these are in the exact same order as in operator == and in the constructor and in the order of fields in the class

Reply:
and in the lerps

Reply:
Fixed up the order of everything and added some missing (yikes) arguments to hashValues/==.